website/docs/user-guides/gitops-configuration.md

```diff
@@ -37,8 +35,9 @@ GitOps enables you to configure a single source of truth for your chaos workflow
 - Some metadata will be pushed to your repository, that is the projectID of your project.
 - Now whenever you schedule a workflow, it will automatically be pushed to your repository. And that repository will be the single source of truth.
 
-## Steps to configure Event-Triggered Chaos Injection
+**Note:** It is also possible to account for the workflows that are directly pushed to the git repository, after configuring GitOps. In this case, if the workflow is a single run workflow, then it starts executing as soon as it is pushed to the repository. Alternatively, if the workflow is a scheduled workflow, then it executes as per the defined schedule.
```

Comment:
Can we specify that "creating" a new workflow in git will execute the workflow whereas "updating" an existing workflow will not execute the workflow rather just update the workflow resource definition in the db and the cluster if applicable?